addons/event/models/event_mail.py

```diff
@@ -120,6 +120,10 @@ def _compute_scheduled_date(self):
 
             scheduler.scheduled_date = date.replace(microsecond=0) + _INTERVALS[scheduler.interval_unit](sign * scheduler.interval_nbr) if date else False
 
+        next_schedule = self.filtered('scheduled_date').mapped('scheduled_date')
```

Comment:
Worth a comment telling that registrations-based schedulers (which are not impacted by this code) are ok because we either send directly communications, either manually trigger the crons in create / write.